Package.swift

```diff
@@ -5,7 +5,7 @@ import PackageDescription
 
 let package = Package(
     name: "NFCPassportReader",
-    platforms: [.iOS(.v13)],
+    platforms: [.iOS(.v13), .macOS(.v10_12)],
```

Comment:
Trengs .macOs også?

Reply:
Nope, but swift packages seems to be designed cross platform and Firebase required a minimun macOS version of 10.12 so I set that to make the build-and-test pipeline pass. Doesn't seem to be a way to exclude a platform.